### PR TITLE
add validations for works collections, generic_works, and generic_files

### PR DESCRIPTION
**Pull request by elrayle**

### CODE REVIEW

**Comment by elrayle** [comment]
This uses the kludge I described for getting multiple types set.  The code will need to be updated when active-triples is updated to allow multiple types.

File: lib/hydra/works/vocab/works_terms.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+require 'rdf'
+module RDFVocabularies
+  class WorksTerms < RDF::Vocabulary("http://hydra.org/works/models#")
```
Comment (jcoyne) [inline lib/hydra/works/vocab/works_terms.rb]: Please switch to `http://projecthydra.org/ns/works/models#` . hydra.org is something else entirely.

**Comment by elrayle** [comment]
Wait to merge pending fuller set of tests.  See https://github.com/projecthydra-labs/hydra-works/issues/77

**Comment by elrayle** [comment]
This is replaced by a cleaner branch...  https://github.com/projecthydra-labs/hydra-works/tree/bring_works_inline_with_pcdm

That branch has failing tests pending completion of the ability for ActiveTriples to support multiple types.

**Pull request closed without merging**